dynacred/src/Defaults.ts

```diff
@@ -0,0 +1,77 @@
+// import { FileIO } from "./FileIO";
```

Comment:
Why is it back? I though you copied/used it in mobile, why exposing it in dynacred?